PotionCraft_SortBookmark/SortBookmark.cs

```diff
@@ -46,7 +46,14 @@ public void SortAllBookmark()
 
             // 所有书签
             List<PotionAndBookmark> all = new List<PotionAndBookmark>();
-            var marks = groupCtl.GetAllBookmarksList();
+
+            // Get a list of all bookmarks before sort for use later
+            var preSortBookmarkList = groupCtl.GetAllBookmarksList();
```

Comment:
Added line to grab a copy of bookmarks before any sorting is done for later use.